diff --git a/git.make b/git.make
index da2302d53..15407d916 100644
--- a/git.make
+++ b/git.make
@@ -9,7 +9,7 @@ Git_Info.C: Git_Info.C.in
 	  else \
 	    echo '#include "ATOOLS/Org/Git_Info.H"' > $@.tmp; \
 	    echo 'static ATOOLS::Git_Info initializer' >> $@.tmp; \
-	    echo '("$(GITTAG)","unknown","unknown","X");' >> $@.tmp; \
+	    echo '("$(GITTAG)","rel-$(PKG_VERSION)","$(PKG_VERSION)","X");' >> $@.tmp; \
 	  fi; \
 	else \
 	  rev=$$(cd $(top_srcdir); git rev-parse HEAD); \
@@ -19,7 +19,7 @@ Git_Info.C: Git_Info.C.in
 	  url=$$(cd $(top_srcdir); git rev-parse --abbrev-ref HEAD); \
 	  echo '#include "ATOOLS/Org/Git_Info.H"' > $@.tmp; \
 	  echo 'static ATOOLS::Git_Info initializer' >> $@.tmp; \
-	  echo '("$(GITTAG)","'$$url'","'$$rev'","X");' >> $@.tmp; \
+	  echo '("$(GITTAG)","rel-$(PKG_VERSION)","$(PKG_VERSION)","X");' >> $@.tmp; \
 	fi; \
 	if test -z $(NOMD5SUM); then \
 	  mds=$$(cat $(addprefix $(srcdir)/, \
